core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -928,6 +928,15 @@ public void stop() {
       internalStop();
    }
 
+   @Override
+   public void stopCache(String cacheName) {
+      try {
+         terminate(cacheName);
+      } catch (Throwable t) {
```

Comment:
Hrmm, I am not sure if we want to catch the exception here as the user explicitly invoked stop on just a single cache. The place this was taken from was an entire stop of the manager, which is a bigger thing.

Is there any reason other than just copying the code over to do this here?

Reply:
Sounds good, that makes sense to me. Yeah, it was only copying the code over.